ts/components/wallet/__test__/OutcomeCodeMessageComponent.test.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from "react";
+import { View } from "react-native";
+import { NavigationParams } from "react-navigation";
+import configureMockStore from "redux-mock-store";
+import { setLocale } from "../../../i18n";
+import { OutcomeCode } from "../../../types/outcomeCode";
+import { renderScreenFakeNavRedux } from "../../../utils/testWrapper";
+import ROUTES from "../../../navigation/routes";
+import { GlobalState } from "../../../store/reducers/types";
+import { applicationChangeState } from "../../../store/actions/application";
+import { appReducer } from "../../../store/reducers";
+import OutcomeCodeMessageComponent from "../OutcomeCodeMessageComponent";
+
+const ASuccessComponent = () => <View testID="a-success-component" />;
+const ASuccessFooter = () => <View testID="a-success-footer" />;
+const onClose = jest.fn();
+
+describe("OutcomeCodeMessageComponent", () => {
+  jest.useFakeTimers();
```

Comment:
Sorry, my fault! 
In the previous comment I meant that all the block in this case is not necessary.
```suggestion

```

Reply:
@debiff If I delete this line, tests gives me this error "ReferenceError: You are trying to `import` a file after the Jest environment has been torn down."

Reply:
Sorry, you're right! The IDE message confused me!